rtb-house/DOCUMENTATION.md

```diff
@@ -2,56 +2,106 @@
 ## General Compatibility
 |Feature|  |
 |---|---|
-| Consent |  |
-| Native Ad Support |  |
-| SafeFrame Support |  |
-| PMP Support | |
+| Consent | Yes |
+| Native Ad Support | No |
+| SafeFrame Support | No |
+| PMP Support | No |
  
 ## Browser Compatibility
 | Browser |  |
 |--- |---|
-| Chrome |  |
-| Edge |  |
-| Firefox |  |
-| Internet Explorer 9 |  |
-| Internet Explorer 10 |  |
-| Internet Explorer 11 |  |
-| Safari |  |
-| Mobile Chrome | |
-| Mobile Safari | |
-| UC Browser | |
-| Samsung Internet | |
-| Opera | |
+| Chrome | Yes |
+| Edge | Yes |
+| Firefox | Yes |
+| Internet Explorer 9 | Yes |
```

Comment:
:D chyba wszystko jedno czy ie9 wspieramy czy nie, ale jak już to chyba ta wersja nie wspierała ajaxów albo była z nimi jakaś straszna zawierucha 
Not sure, w każdym razie

Reply:
Hmm, to jest chyba pokłosie propagandy, która mówi w wymaganiach 'Must support the following browsers: IE 9+, Edge, Chrome, Safari, and Firefox'.
Inni partnerzy (na próbce 1 sprawdziłem, czyli sfalsyfikowałem) którzy mają tylko AJAX zaznaczają obsługę IE9 :P.
Na razie zostawiam.